src/subplots.ts

```diff
@@ -1542,7 +1542,8 @@ export class BasePlot extends PlotData {
   }
 
   protected unpackData(data: any): Map<string, any[]> {
-    const featuresKeys: string[] = Array.from(Object.keys(data.elements[0].values));
+    let featuresKeys: string[] = [];
```

Comment:
Testing truthiness of the first element of a list is a bit weird. We could test the length of it instead : 

```ts
featuresKeys = data.elements.length ? Array.from(Object.keys(data.elements[0].values)) : [];
```

In addition to that, I think a more "english-correct" way of naming a variable that denotes list attribute is to erase the "s" from the first word, ie. `featureKeys` instead of `featuresKeys`

Reply:
Thanks and done